### ISSUE TITLE
Wrong key defined in diagnostics.py

### ISSUE THREAD

**Issue by shawaj** [NebraLtd/hm-diag#502]
https://github.com/NebraLtd/hm-diag/blob/71900ce1b3057dfee8f7aee419b9a1594ae66928/hw_diag/views/diagnostics.py#L262

@kashifpk I believe this should be `KEY_PROVISIONING_KEY` not `SHUTDOWN_GATEWAY_KEY`

I guess we will also need to add a:

```
from hw_diag.diagnostics.provision_key_diagnostic import KEY_PROVISIONING_KEY
```

Can you confirm?

**Comment by shawaj** [comment]
Related to #501

**Comment by kashifpk** [comment]
Yes @shawaj this is wrong, should have been `KEY_PROVISIONING_KEY`. The diagnostic report being used is correct but this would result in incorrect error reporting.
